README.md

```diff
@@ -51,15 +51,15 @@ Done predicting. Predict table : iris.predict
 
 ## How to use SQLFlow
 
-- [Quick Start](doc/quick_start.md)
-- [User Guide](doc/user_guide.md)
-- [Docker Installation](doc/docker_install.md)
+- [Quick Start](/doc/quick_start.md)
+- [User Guide](/doc/language_guide.md)
```

Comment:
User Guide => Language Guide 

btw, what do you think of substituting `Syntax Guide` for `Language Guide`?

Reply:
> User Guide => Language Guide

Sure
> 
> btw, what do you think of substituting `Syntax Guide` for `Language Guide`?

I would suggest still using `Language Guide`, considering SQLFlow extends the SQL(Structured Query **Language**)